src/com/owncloud/android/ui/activity/Uploader.java

```diff
@@ -363,7 +319,7 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         Vector<OCFile> tmpfiles = getStorageManager().getFolderContent(mFile /*, false*/);
         if (tmpfiles.size() <= 0) return;
         // filter on dirtype
-        Vector<OCFile> files = new Vector<OCFile>();
+        Vector<OCFile> files = new Vector<>();
```

Comment:
Annoying Java 7 warning, diamond operator is preferred by the default checker on the right side. I know, I could unset the warning itself. I'm just assuming that there's some real reason by the default warning is that way.